Cvdm.ErrorHandling.Tests/AsyncResultBuilderTests.fs

```diff
@@ -839,6 +839,34 @@ let ``behavior of try-with-finally when thrown`` () =
     test <@ tFinally.Triggered @>
   }
 
+let throwingFn() =
```

Comment:
Unless I'm missing something, I'd like this definition inside the new unit test (preferably just above `Property.check`), since it's only used from there. You're allowed to disagree.